data-processed/EuroCOVIDhub-baseline/metadata-EuroCOVIDhub-baseline.txt

```diff
@@ -0,0 +1,12 @@
+team_name: European COVID-19 Forecast Hub
+model_name: baseline
+model_abbr: EuroCOVIDhub-baseline
+model_contributors: European COVID-19 Forecast Hub team
+website_url: https://covid19forecasthub.eu/
+license: cc-by-4.0
+team_model_designation: primary
+methods: An baseline model against which other models can be evaluated
+team_funding: This model's development is funded by the European Centre for Disease Prevention and Control.
+repo_url: https://github.com/epiforecasts/covid19-forecast-hub-europe
+data_inputs: observed incident cases and deaths (from JHU).
+methods_long: Baseline model automatically generated using past truth data via the script create-baseline.R in the present repository
```

Comment:
```suggestion
methods_long: Baseline model automatically generated using past truth data via the script create-baseline.R in the present repository. Code at: https://github.com/reichlab/covidModels/blob/master/R-package/R/quantile_baseline.R
```

Reply:
Thanks! I did not commit directly your suggestion because `:` don't play well with `yaml` (hence https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/598/commits/1393a2413fcfff152b0dc4dd21f89573b2e75b53).